engine/access/state_stream/backend/backend.go

```diff
@@ -155,6 +157,24 @@ func (b *StateStreamBackend) getExecutionData(ctx context.Context, height uint64
 		return nil, fmt.Errorf("execution data for block %d is not available yet: %w", height, subscription.ErrBlockNotReady)
 	}
 
+	// the spork root block will never have execution data available. If requested, return an empty result.
```

Comment:
this is the second part of the main change. when requesting the root block, return an empty `BlockExecutionData`. this is basically what _would_ exist if ENs produced one for the root block.